### ISSUE TITLE
Incorrect equation/Bug

### ISSUE THREAD

**Issue by joshi994** [wrf-model/WRF#2011]
**Describe the bug**

Module_sf_urban.f which evaluates the building wall temperature (TB) and the road temperature (TG) solving the nonlinear simultaneous equations evaluates the derivative of RB2 (net long wave fluxes absorbed by the building wall with respect to the TG. In the command/line 1303 and 1422 the surface emissivity of the road ($\epsilon_G$), EPSG, is appearing twice. However, the derivative of RB2, DRBDTG2, w.r.t TG in line/command 1313 has EPSG appearing only once. From equation A9 (or B.9 of Lusaka and Kimora, J. of Metro. Society of Japan, Vol 82 No. 1, 67-80, 2004)of the article, Kusaka & Kimora 2004, Journal of Applied Meteorology, it appears that the equation calculating RB2 is wrong.

**To Reproduce**
Use Single Layer Urban Canopy Model (SLUCM) and a compiler depending on the compatibility for testing. 

**Expected behavior**
After correcting the equation, the net longwave fluxes, RB2, absorbed by the building wall should increase as 0 < EPSG < 1 causing an underestimation of the fluxes.  

**Screenshots**
Please refer to the attached correct equation and the command lines in the module.
![Screenshot (425)](https://github.com/wrf-model/WRF/assets/160153650/77fd1c3d-23ad-4071-81d9-2dba405da077)
![Screenshot (426)](https://github.com/wrf-model/WRF/assets/160153650/771fb4ce-221e-4b66-91f2-c0916011a8a3)
![Screenshot (427)](https://github.com/wrf-model/WRF/assets/160153650/e99c9757-2bff-441a-a4e4-815efdccbba0)

**Attachments**
NA.

**Additional context**
NA (the issue is consistent in all versions of WRF.

**Comment by dudhia** [comment]
Might have been better as an actual PR where the difference can be seen more clearly in the review process.

**Comment by dudhia** [comment]
To do a PR you would need to create your own fork first. We have instructions somewhere.

**Comment by weiwangncar** [comment]
@cenlinhe Comment?

**Comment by joshi994** [comment]
@dudhia I will submit a pull request at my earliest. During the WRF tutorial, I was suggested to use the WRF forum but it didn't work so I submitted a new issue.

**Comment by cenlinhe** [comment]
This is indeed a bug. I saw the user already submitted a PR for this issue: https://github.com/wrf-model/WRF/pull/2016
